module/client.js

```diff
@@ -0,0 +1,7 @@
+'use strict';
+const uuid = require('node-uuid');
```

Comment:
@loganlsr best practice is to apply a line break after this statement, to separate your require statements and your module logic